code/game/objects/items/pinpointer.dm

```diff
@@ -0,0 +1,152 @@
+//Pinpointers are used to track atoms from a distance as long as they're on the same z-level. The captain and nuke ops have ones that track the nuclear authentication disk.
```

Comment:
This whole file.

I'm under the impression we already had pinpointers in the code.

This PR doesn't seem to remove any pinpointers, but adds a file full of them.